src/sonic-config-engine/minigraph.py

```diff
@@ -362,6 +362,7 @@ def parse_dpg(dpg, hname):
             vlanid = vintf.find(str(QName(ns, "VlanID"))).text
             vintfmbr = vintf.find(str(QName(ns, "AttachTo"))).text
             vlantype = vintf.find(str(QName(ns, "Type")))
+            vlanmac = vintf.find(str(QName(ns, "MacAddress")))
```

Comment:
nit: I'd move line 365 down right before line 387 for consistency with current retrieval and if checks.

Reply:
Not sure I understand, it seems like almost all the top-level `.find()` calls (for direct children of the `VlanInterface` tag) are made at the start of the loop.

Reply:
I view it differently as `vlantype` was followed by `if vlantype != None:`, `vintf_node` is followed by `if vintf_node` and etc.

Reply:
Oh I see. Changed!